modules/api/src/main/java/org/apache/fluo/api/config/FluoConfiguration.java

```diff
@@ -593,7 +684,7 @@ private int getNextObserverId() {
    *
    * @since 1.1.0
    *
-   * @param className Name of a class that implements {@link ObserverProvider}. Must be non-null and
+   * @param className Name of a class that implements {@link #OBSERVER_PROVIDER}. Must be non-null and
```

Comment:
This link should be left as `{@link ObserverProvider}` which is a link to the class the user should implement.  Need to add a sentence saying it sets the property `{@value OBSERVER_PROVIDER}`.